pkg/utils/gardener/dns.go

```diff
@@ -58,9 +57,9 @@ const (
 )
 
 // GetDomainInfoFromAnnotations returns the provider, domain, and zones that are specified in the given annotations.
-func GetDomainInfoFromAnnotations(annotations map[string]string) (provider string, domain string, zone string, includeZones, excludeZones []string, err error) {
+func GetDomainInfoFromAnnotations(annotations map[string]string) (provider string, domain string, zone string, err error) {
```

Comment:
drop constants in L36 and L39, and their occurrences in example files (YAMLs, ...)?